cmd/daemon.go

```diff
@@ -165,6 +167,7 @@ func init() {
 	daemonCmd.AddCommand(startDaemonCmd)
 	daemonCmd.AddCommand(checkDaemonCmd)
 	startDaemonCmd.Flags().BoolP(gpuEnabledFlag, "g", false, "start daemon with GPU support")
+	startDaemonCmd.Flags().BoolVarP(&kataEnabledFlag, "kata", "k", false, "start daemon inside Kata VM")
```

Comment:
Why are we using `BoolVarP` instead of `BoolP`? We could just retrieve the value just like other flags.

Reply:
BoolP kept giving me compilation issues. I guess there can only be 1 BoolP flag?